### ISSUE TITLE
Installation issue for system cuda version 10.0 with wheels and source 

### ISSUE THREAD

**Issue by aGIToz** [rusty1s/pytorch_geometric#1655]
## 📚 Installation
Hello,
I have been using torch_geometric with torch 1.4cu100 and wanted to use the torch_geo with torch 1.5  onwards.

First I installed the torch_geometric using the provided wheels for cu102 and torch 1.6.0.
This threw the error:
```
OSError: libcusparse.so.10: cannot open shared object file: No such file or directory                                                                        
```
LD_LIBRARY_PATH is fine.
/usr/local/cuda/lib64/ has the libcusparse.so.10.0 and libcusparse.so 

To overcome this I did to install by sources, this time in a venv for torch 1.5.1 with cu101
Then I got the following error:
```
Detected that pytorch and torch_{scatter, sprase, cluster ...} were compiled with different CUDA version. Pytorch has cuda version 10.1 and torch_{scatter, sparse, cluster ...} has cuda version 10.0
```
## Environment

* OS: Linux x86_64
* Python version: 3.6.9
* PyTorch version: Tried with 1.6.0 (installed the wheels) and 1.5.1(installed with sources)
* CUDA/cuDNN version: 10.0
* GCC version: 7.5.0
* How did you try to install PyTorch Geometric and its extensions (wheel, source): Tried both
* Any other relevant information:

## Checklist

- [ Yes ] I followed the [installation guide](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html).
- [Yes ] I cannot find my error message in the [FAQ](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#id1).
- [Nope ] I set up CUDA correctly and can compile CUDA code via `nvcc`. (Actually it is a remote cluster so only sysadmins have the authority to setup cuda )
- [Nope ] I do have multiple CUDA versions on my machine.

## Additional context
So right now **I found a workaround** and I am able to make my old cold (with torch 1.4.0+cu100) run in the  new env (torch1.5.1cu101 with torch_geo installed from sources) by  editing**:
```
if t_major != major or t_minor != minor:
```
to 
```
if t_major!=major
```
In the __init__.py of all the torch_{scatter, sparse, cluster ....}
So that it doesn't check for the minor mismatch. Actually the old code is working so far, but I wonder what are the possible issues I will encounter in future? Is there a better way? Do I need to ask the sysadmins to ugrade the cuda version to atleast 10.1 to avoid this issue ?

**Comment by joc32** [comment]
I face the same problems on PaperSpace and GoogleCloud engine. 

'libtorch_cpu.so: cannot open shared object file: No such file or directory'

**Comment by aGIToz** [comment]
Hey buddy, I am not sure if that could be considered same as this issue.

libtorch_cpu.so means you installed cpu version of pytorch.

But isn't libtorch_cpu.so available inside the python/site-packages/torch/lib/ ?

**Comment by rusty1s** [comment]
The workaround is okay as long as the results are as expected. You can also get around the problem by pointing your CUDA paths to the cudatoolkit==10.1 installed via miniconda.

**Comment by aGIToz** [comment]
Ahh, thanks I will bear this in mind.

**Comment by joc32** [comment]
@aGIToz yes you were right, the system couldn't recognise the correct Torch version. fixed.

**Comment by stonyhu** [comment]
@joc32 Hi, how you solve the 'libtorch_cpu.so' issue? Could you share with me in detail? Face the same problem :(

**Comment by jeongwhanchoi** [comment]
same here, there is same issue on CUDA version 10.0.
